consoles/sshVirtsh.pm

```diff
@@ -399,8 +399,8 @@ sub _copy_image_to_vm_host ($self, $args, $vmware_openqa_datastore, $file, $name
     }
     else {
         $file = $basedir . $file;
-        # requested expected value in GBs, if there is any passed as an argument
-        my $size = $args->{size} // 0;
+        # $args->{size} is expected to be e.g. '20G' but internally we need it as integer
+        my $size = $args->{size} =~ tr/.*G$/.*/dr // 0;
```

Comment:
Now we would get a warning if `$args->{size}` is undef.
`Use of uninitialized value in transliteration (tr///)`

Reply:
I think the `tr` can be simplified:
`tr/G//dr`
`.*` and `$` have no special meaning in transliterations.

Reply:
https://github.com/os-autoinst/os-autoinst/pull/2483